pokemongo_bot/inventory.py

```diff
@@ -108,7 +108,13 @@ class Items(_BaseInventoryComponent):
     STATIC_DATA_FILE = os.path.join(_base_dir, 'data', 'items.json')
 
     def count_for(self, item_id):
-        return self._data[item_id]['count']
+        try:
+            return self._data[item_id]['count']
+        except KeyError:
+            return 0
+
+    def decrement_count(self, item_id):
```

Comment:
With #2482 I added methods to increment/decrement. Also it’s not using raw data from api anymore.
Please update your fork and take a look at it